packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2832,12 +2831,13 @@ export const audiusBackend = ({
     if (isreadSubscribersFromDiscoveryEnabled) {
       // Read subscribers from discovery
       try {
-        const subscribers: APIUser[] = await audiusLibs.User.getUserSubscribers(
-          encodeHashId(userId)
-        )
-        const subscriberIds = subscribers.map((subscriber) =>
-          decodeHashId(subscriber.id)
-        )
+        const encodedUserId = encodeHashId(userId)
+        const bulkResp: { user_id: string; subscriber_ids: string[] }[] =
```

Comment:
So does this return an array of same length as requested user ids?

Reply:
yes is should -- there's a catch in case of an error though